README.md

```diff
@@ -157,8 +157,8 @@ Not yet deployed
 
 > Describe 1 - 3 features you will add to the project.
 
-<!-- - [ ] **Displays scores**
-- [ ] **Submit scores** -->
+- [ ] **Make calculations**
+<!-- - [ ] **Submit scores** -->
 
```

Comment:
- [OPTIONAL] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```